keepsorted/line_group.go

```diff
@@ -475,15 +544,38 @@ func (lg *lineGroup) joinedComment() string {
 }
 
 func (lg *lineGroup) GoString() string {
-	var comment strings.Builder
-	for _, c := range lg.comment {
-		comment.WriteString(fmt.Sprintf("  %#v\n", c))
+	var s strings.Builder
+	s.WriteString("LineGroup{\n")
+	if len(lg.comment) > 0 {
+		s.WriteString("comment=\n")
+		for _, c := range lg.comment {
+			s.WriteString(fmt.Sprintf("  %#v\n", c))
+		}
 	}
-	var lines strings.Builder
-	for _, l := range lg.lines {
-		lines.WriteString(fmt.Sprintf("  %#v\n", l))
+	if len(lg.lines) > 0 {
+		s.WriteString("lines=\n")
+		for _, c := range lg.lines {
+			fmt.Fprintf(&s, "  %#v\n", c)
+		}
+	}
+	if lg.calculated.commentOnly != nil {
+		fmt.Fprintf(&s, "commentOnly=%t\n", *lg.calculated.commentOnly)
+	}
+	if lg.calculated.regexes != nil {
+		for i, regex := range lg.calculated.regexes {
+			if regex.wasUsed() {
+				fmt.Fprintf(&s, "regex[%d]=%#v\n", i, regex)
+			}
+		}
+	}
+	if lg.calculated.lines != "" {
```

Comment:
I get that it's for debugging, but doesn't this make it quite noisy since it would repeat the lines and comments twice? Seems like that's useful in some cases but might be a lot of output in others. Makes me wonder if it should be behind a flag or the likes...